packages/react-router-dom/index.tsx

```diff
@@ -216,7 +218,10 @@ export function createBrowserRouter(
 ): RemixRouter {
   return createRouter({
     basename: opts?.basename,
-    future: opts?.future,
+    future: {
+      ...opts?.future,
+      v7_prependBasename: true,
```

Comment:
Always turn this on for RR - it used to be RR doing the prepending, now it's the router